src/main/java/net/hypixel/nerdbot/util/skyblock/Stat.java

```diff
@@ -35,6 +35,8 @@ public enum Stat {
     OVERFLOW_MANA("ʬ", "Overflow Mana", MCColor.DARK_AQUA),
     REQUIRE("❣",  "Requires", MCColor.RED, StatColorParser::postStatColorParser),
     RECIPE("", "Right-click to view recipes!", MCColor.YELLOW, StatColorParser::noParsing),
+    COOP_SOULBOUND("", "\\n\\n* Co-op Soulbound *", MCColor.DARK_GRAY, StatColorParser::noParsing),
```

Comment:
Replacements likely shouldn't have newlines in them- that should be the requirement of the user to put in

Reply:
okay that's fair I'll change it